docs/docs/sourcing-from-drupal.md

```diff
@@ -2,7 +2,83 @@
 title: Sourcing from Drupal
 ---
 
-This is a stub. Help our community expand it.
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+### Why use Drupal + Gatsby together?
+
+Using Drupal as a headless CMS with Gatsby is a great way to get an enterprise-quality CMS for free, paired with a great modern development experience and all the benefits of the JAMstack, like performance, scalability, and security.
+
+It only takes a few steps to use Gatsby with Drupal as a headless CMS (also known as decoupled Drupal).
+
+[A delicious Gatsby + Drupal site](https://using-drupal.gatsbyjs.org/)
+
+### How to implement Drupal + Gatsby
+
+Hooking up Gatsby to a new or existing Drupal site takes just a few steps:
+
+- Add the `gatsby-source-drupal` plugin to your Gatsby site
+- Configure the plugin with your Drupal credentials & hosted URL
+- Access all your Drupal data using GraphQL queries
+
+An example code is given below :
+
+    module.exports = {
```

Comment:
It's more common on Gatsby docs to put that into code block (three back-ticks).

Insert it in:

````
```javascript:title=gatsby-config.js
// stuff goes here
```
````

And everything except the `gatsby-source-drupal` is not relevant to the tutorial, so you can remove that 👍